src/Validator.php

```diff
@@ -226,7 +226,9 @@ private function getNumericRepresentation($letterRepresentation)
     private function local_bcmod($operand, $modulus)
     {
         if (function_exists('bcmod')) {
-            return bcmod($operand, $modulus);
+            return PHP_VERSION_ID >= 70200
```

Comment:
`>` right?

Reply:
Its more inclusive this way 7.2.0 is also matched. Even though its unlikely that anybody uses it.

Reply:
Sorry, what I meant was that this should only be executed for PHP +7.3 and not 7.2. But of course the correct comparison would have been `PHP_VERSION_ID >= 70300`.

Reply:
bcscale was changed on 7.3+ but this fixes bcmod (7.2)

Reply:
The `scale` argument of the `bcmod` function was introduced in `PHP 7.2.0`.
So `PHP_VERSION_ID >= 70200` looks good to me.

@xabbuh Or did I miss something?

Reply:
Looks good to me. I missed that the patch was changed to not use `bcscale()` anymore.